.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: vim
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
 
-install: |
-  sudo apt-get update
-  sudo apt-get install vim
+before_script: |
   cd ..
+  if [ $(ruby -e 'puts RUBY_VERSION') == '1.8.7' ]; then
```

Comment:
Minor things: use quotes for the expansion as `"$( something )"` , and use `=` instead of `==` as it is POSIX.